qsirecon/tests/run_local_tests.py

```diff
@@ -59,30 +70,45 @@ def run_command(command, env=None):
 
     if process.returncode != 0:
         raise RuntimeError(
-            f"Non zero return code: {process.returncode}\n" f"{command}\n\n{process.stdout.read()}"
+            f"Non zero return code: {process.returncode}\n"
+            f"{command}\n\n{process.stdout.read()}"
         )
 
 
-def run_tests(test_regex, test_mark):
+def run_tests(test_regex, test_mark, check_path):
     """Run the tests."""
     local_patch = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-    mounted_code = "/usr/local/miniconda/lib/python3.10/site-packages/qsirecon"
-    run_str = "docker run --rm -ti "
-    run_str += f"-v {local_patch}:{mounted_code} "
-    run_str += "--entrypoint pytest "
-    run_str += "pennlinc/qsirecon:unstable "
-    run_str += (
-        f"{mounted_code}/qsirecon "
-        f"--data_dir={mounted_code}/qsirecon/tests/test_data "
-        f"--output_dir={mounted_code}/qsirecon/tests/pytests/out "
-        f"--working_dir={mounted_code}/qsirecon/tests/pytests/work "
-    )
-    if test_regex:
-        run_str += f"-k {test_regex} "
-    elif test_mark:
-        run_str += f"-rP -o log_cli=true -m {test_mark} "
+    mounted_code = "/opt/conda/envs/qsiprep/lib/python3.10/site-packages/qsirecon"
+
+    if check_path:
+        run_str = (
+            "docker run --rm -ti "
+            "--entrypoint /bin/ls "
+            f"pennlinc/qsirecon:unstable {mounted_code}"
+        )
+        try:
+            run_command(run_str)
+            print(f"Path found: {mounted_code}.")
+        except RuntimeError:
+            raise FileNotFoundError(f"Path not found: {mounted_code}")
+
+    else:
+        run_str = (
+            "docker run --rm -ti "
```

Comment:
Roadmap: we'll need to add some more mounts for freesurfer input and custom atlases